mail_activity/README.rst

```diff
@@ -0,0 +1,65 @@
+==========
+Activities
+==========
+
+This module backports the activities from 11.0.
+
+Configuration
+=============
+
+To configure this module, you need to do the same as in 11.0.
+
+Usage
+=====
+
+To use this module, you need to do the same as in 11.0.
+
+Known issues / Roadmap
+======================
+
+* The button 'Mark done' that appears for each activity in the chatter
+  is not available.
+
+* It is not possible to schedule a next activity based on a given activity.
+
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/social/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+**This module is a backport from Odoo SA and as such, it is not included in the OCA CLA. That means we do not have a copy of the copyright on it like all other OCA modules.**
```

Comment:
Guys, do you have guidelines somewhere related to porting code from Odoo SA (or other third-party code)? The GPL licenses require that you identify specifically in each file what code is covered by what copyright and what license (when not uniform).

I find the current ports a little confusing, with a mix of licensing and copyright info. I see no copyright notice from Odoo SA (and its third-party contributors), and I can't tell by reading the source files who owns the copyright on what part of the code, nor what you changed (perhaps I simply missed it?)

For the record, the short notice "# Part of Odoo ..." only works inside the official Odoo code because:
 - it comes with matching LICENSE and COPYRIGHT files
 - external contributions are always subject to a CLA and authorship attribution is done via Git metadata on commits.
When you copy the code outside of official branches you can't replicate all of these conditions AFAICT, so you need to provide proper attribution, license and copyright info in each file, in context, and identify all your changes.

I know this stuff can be a PITA, but it's a key principle of open source, and I know we all want to do it right :-)

Reply:
@odony AFAIK, the module is full licensed as LGPL-3, which matches your license, isn't it?

Reply:
@pedrobaeza well, there seems to be some conflicting info about that too, even in the manifest file ;-)
https://github.com/Eficent/social/blob/c3c1af5a6c02f80cad5158d3f6914311eebeb96b/mail_activity/__openerp__.py#L3

Reply:
@odony I totally agree with you. I will add Odoo, S.A. wherever the code was moved, and change the licensing type. This feature is super useful, and unfortunately not all companies are able to move to 11.0 as quick as we'd like, so backporting is sometimes an option.

Reply:
You are right, @odony. Some copy/paste error, but @jbeficent is on it as he has said. Thanks for noticing.